security/hosts_multitenancy.adoc

```diff
@@ -0,0 +1,109 @@
+mandatory access controls (MAC) for every user, application, process, and file.
+SELinux is the brick wall that is going to stop you if you manage to break out
+of (accidentally or on purpose) the namespace abstraction.
+
+- _CGroups_ (or control groups) limit, account for, and isolate the resource usage
+(CPU, memory, disk I/O, network, etc.) of a collection of processes. CGroups are
+used to ensure your container will not impacted by another container on the same
+host.
+
+- Finally, a _secure computing mode (seccomp)_ profile can be associated with a
+container to restrict available system calls.
+
+You can further enhance security for your applications and infrastructure by
+deploying your containers to RHEL Atomic Host. RHEL Atomic Host reduces the attack
+surface by minimizing the host environment and tuning it for containers.
+
+As a proof point of the security features available to you with RHEL and RHEL
+Atomic Host, RHEL 7.1 recently received Common Criteria certification, including
+certification of the Linux Container Framework Support.
+
+[discrete]
+[[security-hosts-further-reading-1]]
+==== Further Reading
+
+- Linux man page: link:http://man7.org/linux/man-pages/man7/namespaces.7.html[_namespaces(7)_]
+- _Red Hat Enterprise Linux Atomic Host Overview of Containers in Red Hat Systems_:  link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/overview_of_containers_in_red_hat_systems/introduction_to_linux_containers#secure_containers_with_selinux[Secure Containers with SELinux]
+- _Red Hat Enterprise Linux Resource Management Guide_:  link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Resource_Management_Guide/chap-Introduction_to_Control_Groups.html[Introduction to Control Groups (CGroups)]
+- _Red Hat Enterprise Linux Atomic Host Container Security Guide_: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/container_security_guide/linux_capabilities_and_seccomp[Linux Capabilities and seccomp]
+- Kernel documentation: link:https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt[_seccomp_]
+
+[[security-host-multi-tenancy]]
+== Multi-tenancy: Virtualization Versus Containers
+
+Traditional virtualization also enables multi-tenancy, but in a very different
+way from containers. Virtualization relies on a hypervisor spinning up guest
+virtual machines (VMs), each of which has its own operating system (OS), as well
+as the running application and its dependencies.
+
+With VMs, the hypervisor isolates the guests from each other and from the host
+kernel. Fewer individuals and processes have access to the hypervisor, reducing
+the attack surface on the physical server. That said, security must still be
+monitored: one guest VM may be able to use hypervisor bugs to gain access to
+another VM or the host kernel. And, when the OS needs patching, it must be
+patched on all guest VMs using that OS.
```

Comment:
We need to think about how https://github.com/kubevirt may change the way we message this, or double down on the messaging for areas such as this (Security).